README.md

```diff
@@ -4,3 +4,119 @@
+    This rule maps MQTT topics of the form "building/{building}/room/{room}/#" to the Kafka topic "building_{building}room{room}". The wildcard "#" represents one or more levels in the topic hierarchy. For example, if the MQTT topic is "building/A/room/1/floor/2", it will be mapped to the Kafka topic "building_A_room_1_floor_2".
+
+2. MQTT Topic: "sensors/#" -> Kafka Topic: "sensor_others"
+
+    This rule maps any MQTT topic starting with "sensors/" followed by any number of levels in the hierarchy to the Kafka topic "sensor_others". The "#" wildcard matches any number of levels. For example, if the MQTT topic is "sensors/temperature/living-room", it will be mapped to the Kafka topic "sensor_others".
+
+3. MQTT Topic: "sensors/+/data" -> Kafka Topic: "sensor_data"
+
+    This rule maps MQTT topics of the form "sensors/+/data" to the Kafka topic "sensor_data". The "+" wildcard matches a single level in the topic hierarchy. For example, if the MQTT topic is "sensors/temperature/data", it will be mapped to the Kafka topic "sensor_data".
+
+These mapping rules demonstrate how different MQTT topics can be mapped to corresponding Kafka topics based on predefined patterns. The MQTT Bridge uses these rules to map MQTT topics to Kafka topics.
+The order in which the rules are defined is important. The MQTT Bridge will use the first rule that matches the MQTT topic. For example, if the MQTT topic is "sensors/temperature/data", 
+it will be mapped to the Kafka topic "sensor_data" because the third rule matches the MQTT topic. If the third rule was not defined, the MQTT Bridge would use the second rule to map the MQTT topic to the Kafka topic "sensor_others".
+
+### 2. MQTT Bridge Configuration
+The user can configure the MQTT Bridge using the `application.properties` file. This section describes the configuration properties that can be used to configure the MQTT Bridge.
+The MQTT bridge can be configured using the appropriate prefix. Example:
+- `bridge.` is the prefix used for general configuration of the Bridge.
+- `mqtt.` is the prefix used for MQTT configuration of the Bridge.
+- `kafka.` is the prefix used for Kafka configuration of the Bridge.
+### 2.1 Procedure to configure the MQTT Bridge
+1. Find the `application.properties` file in the `config` directory.
+2. Edit the file to configure the MQTT Bridge.
+    A valid configuration file should look like this:
+    ```properties
+    # Bridge configuration
+    bridge.id=my-bridge
+    # MQTT configuration
+    mqtt.server.host=localhost
+    mqtt.server.port=1883
+    # Kafka configuration
+    kafka.bootstrap.servers=localhost:9092
+    ```
+3. Save the file.
+
+The following table describes the configuration properties that can be used to configure the MQTT Bridge.
+
+| Setting                        | Description                                     | E.g. Value              |
+| ------------------------------ | ----------------------------------------------- |-------------------------|
```

Comment:
I don't think we want an e.g. Value column, but a "Default" column only.

Reply:
@ppatierno I agree.